### ISSUE TITLE
Publishing open access at low cost: what are our options?

### ISSUE THREAD

**Issue by kipkurui** [BioinfoNet/OpenScienceKEHackathon#2]
Create a resource that can be used to guide students and early career researchers in Africa to publish open access, and at subsidized or waived article processing charge.

In most Kenyan Institutions, students are required to publish at least one paper at Msc level. The common question is: where do we publish at low cost? To help students and early career scientists take decisions on where to publish, we intend to create an open resource which provides basic information from the journal: APC, whether there is a waiver for Kenyan students, etc.

**Comment by Shuyib** [comment]
How about https://www.biorxiv.org/ ?

**Comment by kipkurui** [comment]
Biorxiv is a pre-print server.  That's a good place to make research accessible as it awaits peer review. The idea here is it create a list of journals that are friendly to students and ECR.  We'll include list of pre-prints as well

**Comment by karegapauline** [comment]
Can we talk about elife? Then also advise on how they can get the fee
waivered? I've seen you can request for waivered fee for certain journals
like PLOS and Hindawi.

**Comment by kipkurui** [comment]
@karegapauline Yes, those are some of the journals we should include in the list. And, yes, that's the plan, to create a one-stop place where students can get information on where to publish at a low or zero cost.

**Comment by esohkevin** [comment]
You may wanna check out the 'Directory of Open Access Journals' via this link https://doaj.org/
It is a service of 'Infrastructure Services for Open Access' https://is4oa.org/

**Comment by kipkurui** [comment]
@esohkevin That is a great place to start, thanks.

**Comment by SylvieVullioud** [comment]
Please consider future [JPSS ](https://www.journalquality.info/en/ ) evaluation South journals. Currently, African Online Journals ([African JOL](https://www.ajol.info/) ) are assessed by independant consultant for quality. JPSS is currently finalist of prestigious award ALPSP award... https://www.alpsp.org/Awards 

The current list of JPSS evaluated journals is [here ](https://www.journalquality.info/en/journals-all/). Maybe one is suiting you? Or if you can wait until end of year 2018, African JOL will be assessed and result published? 

Hindawi (originally Egypta now London based publisher) may be a good option too?

**Comment by kipkurui** [comment]
Thank you @SylvieVullioud for your input. We will definitely consider those. @karegapauline Keep an eye on [this twitter thread](https://twitter.com/Calkibet/status/1032963084028141569) for more suggestions.

**Comment by karegapauline** [comment]
Working on it Caleb. I've seen it. We should be able to maintain the
resource in a way that we make changes as soon as they come up.

**Comment by amelgh** [comment]
There is also a "new" African preprints server: https://osf.io/preprints/africarxiv

**Comment by kipkurui** [comment]
Thanks, @amelgh. We could consider of submitting a pre-print from our collaborative writing there

**Comment by esohkevin** [comment]
That's right. Looks great

**Comment by kipkurui** [comment]
Thank you @all for your contribution to this idea. The resource created from this issue can be found [here](https://github.com/BioinfoNet/Openaccess-Options). You continued support is highly appreciated.
